src/main/java/be/seeseemelk/mockbukkit/inventory/ItemStackMock.java

```diff
@@ -184,6 +184,17 @@ public void addUnsafeEnchantment(@NotNull Enchantment ench, int level)
 		}
 	}
 
+	@Override
+	public int getEnchantmentLevel(Enchantment ench)
```

Comment:
Although there's little to test here, could you just add a test that triggers this method, and checks if it does anything? (in ItemStackMockTest)

Reply:
Alright